src/globus_cli/commands/flows/run/resume.py

```diff
@@ -92,8 +93,15 @@ def check_inactive_reason(
     # (consents may be present without session requirements met)
     if unhandled_requirements:
         raise CLIAuthRequirementsError(
-            "This timer needs strong authentication in order to resume.",
+            "This run needs strong authentication in order to resume.",
             gare=gare,
+            epilog=textwrap.dedent(
+                f"""\
+                After updating your session, resume the timer with
```

Comment:
I think two of my comments were resolved by a later commit. This one is still relevant, though.

Reply:
I now realize that I missed this one because it's in the `epilog` and I was checking the other messages when I did the fixing. I just came back and got this one done too.